sdk/src/account.rs

```diff
@@ -45,6 +45,14 @@ impl Account {
             executable: false,
         }
     }
+
+    pub fn deserialize_data<T: serde::de::DeserializeOwned>(&self) -> Result<T, bincode::Error> {
```

Comment:
what's the value of placing these in Account?  how come they don't have the names state() and set_state() ?

Reply:
The names reflect that there is no assumptions about what's being deserialized, other than that the data is encoded with `bincode`.  We can move this out of Account, but I'm okay with Account having a bincode dependency (and not okay with a dependency on anything that's specific to instruction processing, like calling it "state()" or "contract()").

Reply:
Also, when using that property naming convention `state()` vs `do_stuff()`, you might be implying it's "free". The malloc for a DeserializeOwned is not where we should be using that abstraction.